generate-release-notes.js

```diff
@@ -3,6 +3,8 @@ const execFile = util.promisify(require('child_process').execFile);
 
 const repoURL = process.env.GITHUB_SERVER_URL + '/' + process.env.GITHUB_REPOSITORY;
 
+const excludePreset = /^meta|^document|^lint|^refactor|readme|dependencies|^\d\.\d\.\d/i;
```

Comment:
Last time I was working on it, this `\d\.\d\.\d` pattern does not work.

See https://github.com/notlmn/release-with-changelog/blob/exclude-preset/index.js#L6 for complete pattern.

Reply:
This is a proper regex, not a string

Reply:
Ah, I should keep reading code properly.

Maybe?

```suggestion
const excludePreset = /^meta|^document|^lint|^refactor|readme|dependencies|^v?\d+\.\d+\.\d+/i;
```